html/changelogs/Eyeveri_Noob_ScienceRemap.yml

```
@@ -0,0 +1,51 @@
+################################
+# Example Changelog File
+#
+# Note: This file, and files beginning with ".", and files that don't end in ".yml" will not be read. If you change this file, you will look really dumb.
+#
+# Your changelog will be merged with a master changelog. (New stuff added only, and only on the date entry for the day it was merged.)
+# When it is, any changes listed below will disappear.
+#
+# Valid Prefixes:
+#   bugfix
+#   wip (For works in progress)
+#   tweak
+#   soundadd
+#   sounddel
+#   rscadd (general adding of nice things)
+#   rscdel (general deleting of nice things)
+#   imageadd
+#   imagedel
+#   maptweak
+#   spellcheck (typo fixes)
+#   experiment
+#   balance
+#   admin
+#   backend
+#   security
+#   refactor
+#################################
+
+# Your name.
+author: ReadThisNamePlz, 50%Noob
+
+# Optional: Remove this file after generating master changelog.  Useful for PR changelogs that won't get used again.
+delete-after: True
+
+# Any changes you've made.  See valid prefix list above.
+# INDENT WITH TWO SPACES.  NOT TABS.  SPACES.
+# SCREW THIS UP AND IT WON'T WORK.
+# Also, all entries are changed into a single [] after a master changelog generation. Just remove the brackets when you add new entries.
+# Please surround your changes in  double quotes ("), as certain characters otherwise screws up compiling. The quotes will not show up in the changelog.
+changes:
+  - maptweak: "Remapped the entirety of the Science department on Deck Two."
+  - maptweak: "Circuits and RnD expanded."
+  - maptweak: "Science entrance moved."
+  - maptweak: "Xenobotany and Xenobiology received layout changes."
+  - maptweak: "Telescience was located to a different area."
+  - maptweak: "Exploratory Chemistry is now in its own secure area."
+  - maptweak: "AI Upload Foyer made smaller."
+  - maptweak: "AI Chamber altered slightly."
+  - maptweak: "Research Directors Office tweaked for QOL"
+  - maptweak: "Science now has its own departmental entrance to a secure wing."
+  - maptweak: "Edits the Deck one ladder to the right by one tile"
```

```suggestion
  - maptweak: "Remapped the entirety of the research department on deck two."
```
one entry works fine.